### DISCUSSION TITLE
Thank you very much.

### DISCUSSION THREAD

**Discussion by Vilhelm-Ian** [mortii/anki-morphs#112] [General]
Thank yu very much mortii for the fork. In it's current state for me it's better in every way than the original. For me it has 1x1 feature parity. With a better design and better performance. Plus some additional features.

I am currently not employed. Can you share something like buy me a coffee or a cryptocurrency public address. So when I can send you a small amount.

Also happy new year

**Comment by mortii** [comment]
Thank _you_, you have been a tremendous help and you have motivated me to work on this project. I don't want any money, thank you for offering though :)

Happy new year, and I hope you succeed beyond your wildest dreams! :tada:
